zerver/lib/export.py

```diff
@@ -1699,3 +1699,17 @@ def export_realm_wrapper(realm: Realm, output_dir: str,
         os.remove(tarball_path)
         print("Successfully deleted the tarball at %s" % (tarball_path,))
     return public_url
+
+def get_public_exports_serialized(user: UserProfile) -> List[Dict[str, Any]]:
+    all_exports = RealmAuditLog.objects.filter(realm=user.realm,
+                                               event_type=RealmAuditLog.REALM_EXPORTED)
+    exports_dict = {}
+    for export in all_exports:
+        exports_dict[export.id] = dict(
+            id=export.id,
+            realm=export.realm.name,
+            event_time=export.event_time.ctime(),
+            acting_user=user.full_name,
```

Comment:
This is wrong -- `user` here is the currently logged-in user; you want `export.acting_user.id` (the frontend should be able to map that to a name).

Reply:
Fixed!